client/src/common/organization.ts

```diff
@@ -0,0 +1,8 @@
+// Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS
+
+export function generateTrialOrganizationName(userEmail: string): string {
+  const timestamp = new Date().valueOf().toString();
+  const part = userEmail.slice(0, 11).replaceAll(/[^a-zA-Z0-9-_]/g, '_');
```

Comment:
We talked about taking only the prefix of the mail (before the @) AND limited to a certain length.
Missing the first part here.